ehitchhiking-ui/src/app/shared/components/blacklist/blacklist.component.html

```diff
@@ -0,0 +1,16 @@
+<h1 mat-dialog-title class="blacklist-title">Blacklist</h1>
+<div mat-dialog-content class="blacklist">
+	<div
+		class="blacklist-user"
+		*ngFor="let blacklistuser of blacklistUsersArray; let i = index"
+	>
+		<i class="material-icons profile-photo">account_circle</i>
+		<p class="blacklist-user-name">{{ blacklistuser }}</p>
+		<i class="material-icons remove-icon" (click)="removePerson(i)"
+			>remove_circle</i
+		>
+	</div>
+</div>
+<div mat-dialog-actions class="blacklist-back-btn">
+	<button mat-button (click)="back()">Back</button>
```

Comment:
bad naming: name of function should be a verb

Reply:
renamed